datakyt/db/test_insert_data_in_project_table.py

```diff
@@ -0,0 +1,46 @@
+import psycopg2
+import csv
+import os
+import unittest
+
+
+class TestDB(unittest.TestCase):
+
+    @staticmethod
+    def insert_csv_file_to_database(path, conn):
```

Comment:
what is this?

Reply:
This is the function in which I write information from the file to the database. (I added the documentation.)

Reply:
remove it, import the implemented one